packages/client/components/PokerActiveVoting.tsx

```diff
@@ -100,8 +100,11 @@ const PokerActiveVoting = (props: Props) => {
   const {id: stageId, scores} = stage
   const hasVotes = scores.length > 0
   const isFacilitator = viewerId === facilitatorUserId
-  const viewerHasVoted = Boolean(scores.find(({userId}) => userId === viewerId))
-  const checkedInCount = meetingMembers.length
+  const viewerHasVoted = useMemo(() => !!scores.find(({userId}) => userId === viewerId), [scores])
```

Comment:
AFAIK sometimes using `useMemo` is more expensive than not using it.
that said, it doesn't matter much either way, so typically if the arrays are < 100 I just don't memoize things.
but IIRC, you had a blog post that was talking about this? would love to learn more

Reply:
True! Adding `useMemo` adds a function call which makes the performance worse when arrays are small, plus it adds extra complexity. Removed!